gems/aws-sdk-lookoutmetrics/lib/aws-sdk-lookoutmetrics/plugins/content_type.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Aws
+  module LookoutMetrics
+    module Plugins
+      class ContentType < Seahorse::Client::Plugin
+
+        def add_handlers(handlers, _config)
+          handlers.add(Handler)
+        end
+
+        class Handler < Seahorse::Client::Handler
+          def call(context)
+            # Some operations break when given an empty content-type header.
+            # The SDK adds this blank content-type header
+            # since Net::HTTP provides a default that can break services.
+            # We're setting one here even though it's not used or necessary.
```

Comment:
comment nit pick for this middleware:  It is necessary right?

Reply:
I copied it from existing plugin.. I think it's still relevant.